ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/documentation/management-tech-support.j2

```diff
@@ -0,0 +1,40 @@
+{# Management Tech-Support #}
+{% if management_tech_support is arista.avd.defined %}
+
+## Management Tech-Support
+
+{%   if management_tech_support.policy_show_tech_support is arista.avd.defined %}
+### Policy
+
+{%     if management_tech_support.policy_show_tech_support.exclude_commands is arista.avd.defined %}
+#### Exclude Commands
+
+| Command | Type |
+| ------- | ---- |
+{%       if management_tech_support.policy_show_tech_support.exclude_commands is arista.avd.defined %}
+{%         for exclude_command in management_tech_support.policy_show_tech_support.exclude_commands %}
```

Comment:
You have provided wrong indentation (2 spaces). Should be 4 spaces at every next level. Please check indentation in all templates.